sms_939/models/sms_sender.py

```diff
@@ -0,0 +1,66 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Copyright (C) 2018 Compassion CH (http://www.compassion.ch)
+#    Releasing children from poverty in Jesus' name
+#    @author: Nathan Fluckiger <nathan.fluckiger@hotmail.ch>
+#
+#    The licence is in the file __manifest__.py
+#
+##############################################################################
+from odoo import models, fields, api
+from odoo.tools.config import config
+import httplib
+import base64
+import urllib
+
+
+class SmsSender(models.TransientModel):
+
+    _name = 'sms.sender'
+
+    text = fields.Text()
+    partner_id = fields.Many2one(comodel_name='res.partner',
+                                 compute='_compute_partner')
+
+    @api.model
+    @api.depends('text')
+    def _compute_partner(self):
+        self.partner_id = self.env.context.get('partner_id', False)
+        if not self.partner_id:
+            print("Une erreur est survenue ")
+
+    @api.multi
+    def send_sms(self):
+        if not self.partner_id.mobile:
```

Comment:
You can start your method with `self.ensure_one()` to show this method will always be called on a single record.